### PR TITLE
fix typo etc

### PR DESCRIPTION
**Pull request by jeremylenz**
## Updated Packages

* [x] root
* [ ] @theforeman/vendor
* [ ] @theforeman/vendor-dev
* [ ] @theforeman/vendor-core

## PR Type

* [x] Bugfix
* [x] Feature
* [ ] Code style update (whitespace, formatting, missing semicolons, etc.)
* [ ] Refactoring (no functional changes, no api changes)
* [ ] Build related changes
* [ ] CI related changes
* [ ] Documentation content changes
* [ ] Other… Please describe:

## Does this PR introduce a breaking change?

* [ ] Yes
* [x] No

## Does this PR fixes open issues?

* [ ] Yes
* [x] No

## Do you use this PR in another repository?

* [ ] Yes
* [x] No

### CODE REVIEW

File: packages/vendor-core/package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
   ],
   "dependencies": {
     "@novnc/novnc": "^1.0.0",
+    "@patternfly/react-icons": "^3.14.2",
```
Comment (sharvit) [inline packages/vendor-core/package.json]: Any reason to add this new dep? looks like a leftover

Comment (jeremylenz) [reply packages/vendor-core/package.json]: @sharvit This PR is just so I could test your other PR; it's not meant to be merged. :smile:

Comment (sharvit) [reply packages/vendor-core/package.json]: Haha thanks!

**Comment by sharvit** [comment]
Notice the commit need to be formatted with `npm run commit` using `fix` or `feature` so it will auto-deploy.

**Comment by sharvit** [comment]
Thank you for the help here @jeremylenz, closing since this deploy method isn't going to work.

**Pull request closed without merging**